src/util.cpp

```diff
@@ -484,6 +484,24 @@ const boost::filesystem::path &ZC_GetParamsDir()
     return path;
 }
 
+// Return the user specified export directory.  Create directory if it doesn't exist.
+// If user did not set option, or if the path is not a directory, return an empty path.
```

Comment:
Rebased. Changed behaviour so that if the user did not set an option, an empty path is returned. An exception is thrown if there is a filesystem error.